.github/workflows/test-all.yml

```diff
@@ -258,7 +265,7 @@ jobs:
       - name: "Download every job status"
         uses: elonh/download-artifact-regexp@master # FIXME/TODO: Switch to actions/download-artifact once they officially support wildcards, see https://github.com/actions/download-artifact/issues/6
         with:
-          pattern: reports-*
+          pattern: test-status-${{ github.event.inputs.which-gap }}-*
```

Comment:
I had the same / similar idea in PR #225 :-). I can close that one if you are fixing it here, though.